packages/elements/src/components/HelpGuide/__styles__/styles.ts

```diff
@@ -1,6 +1,7 @@
 import { css } from 'linaria'
 
-export const fadeStyle = css`
+export const helpGuildeContentStyle = css`
```

Comment:
Can we think about the  unique class name convention? Some end with style, while some end when component name?

Reply:
I agree with @phmngocnghia on this one. We should decide, happy for someone to propose something. Remember also, when we output the style it gets dashed to make it developer friendly eg help-guide-content-style.

I would suggest `<<component-name>>-<<block>>-<<modifier>>` would work.

Also you have a typo - Guide not Guilde!